### ISSUE TITLE
a11y: Help buttons can't be navigated with keyboard

### ISSUE THREAD

**Issue by tahini** [chairemobilite/evolution#241]
The button circled in the screenshot cannot be reached when navigating with tabs

![Capture d’écran du 2023-09-07 08-51-18](https://github.com/chairemobilite/evolution/assets/4398897/bf99fa03-c299-4cc5-b8b0-efa98bf418c1)

**Comment by millette** [comment]
Sans doute à cause de https://github.com/chairemobilite/evolution/blob/main/packages/evolution-frontend/src/components/survey/widgets/HelpPopupLink.tsx#L38

```jsx
<button
  type="button"
  className="button helper-popup blue small"
  onClick={() => setModalIsOpened(true)}
  tabIndex={-1}
>
  <FontAwesomeIcon icon={faQuestionCircle} className="faIconLeft" />
  {props.title}
</button>
```

Je peux retirer le tabIndex pour qu'il retrouve sa place dans le *flow*.
